### PR TITLE
LRQA-70392, LRQA-70564 Fix for task grouping and meaningful test description

### PR DESCRIPTION
**Pull request by weller0246**
https://issues.liferay.com/browse/LRQA-70392
https://issues.liferay.com/browse/LRQA-70564

### CODE REVIEW

**Comment by weller0246** [comment]
ci:test:sf

**Comment by weller0246** [comment]
ci:test:security:nocompile

**Comment by weller0246** [comment]
ci:test:sf

**Comment by weller0246** [comment]
ci:test:sf

**Comment by weller0246** [comment]
ci:test:security:nocompile

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]
Hi @weller0246 , could you please commit the description changes into a separate commit?

Thanks in advance!

**Comment by weller0246** [comment]
ci:test:sf

**Comment by weller0246** [comment]
ci:test:security:nocompile

**Comment by weller0246** [comment]
@ferenconodiD @gaborlovasdotcom  I separated the two tickets, please review again  if you have the time so we can forward it to Bchan :)

**Review by ferenconodiD — APPROVED** [review]
LGTM!

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase [hunk 1]
```diff
@@ -22,7 +22,7 @@ definition {
 		}
 	}
 
-	@description = "This is a use case for LPS-129398. Remember device with multiple auth without session."
+	@description = "This is a use case for LPS-129398. User can authorize external application multiple times without any user session, when remember device is chosen."	@priority = "4"
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase:25]: Please remove @priority 4 from the end of the line!
cc @ferenconodiD

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase [hunk 2]
```diff
@@ -236,7 +236,7 @@ definition {
 		}
 	}
 
-	@description = "This is a use case for LPS-129397 TC-2: Revoke multiple authorization via new kebab menu option in OAuth2 application's row when application is trusted."
+	@description = "This is a use case for LPS-129397 TC-2: User can revoke authorizations from a trusted application via kebab menu on the OAuth2 administration page."	@priority = "3"
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase:239]: Please remove the priority from the description
cc @ferenconodiD

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase [hunk 3]
```diff
@@ -1187,7 +1187,7 @@ definition {
 		}
 	}
 
-	@description = "This is a use case for LPS-129395. View external option remember device."
+	@description = "This is a use case for LPS-129395. Remember device option should be visible in Oauth2 admin page if it was set in the application settings."
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase:1190]: Please use the official letters in descriptions: Oauth2 -» OAuth 2
cc @ferenconodiD

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase [hunk 4]
```diff
@@ -1215,7 +1215,7 @@ definition {
 		}
 	}
 
-	@description = "This is a use case for LPS-129398. Remember device with multiple auth."
+	@description = "This is a use case for LPS-129398. User can authorize multiple external applications which have the remember device option checked"
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase:1218]: Please end the description sentence with a dot "." to be more consistent.
cc: @ferenconodiD

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase [hunk 5]
```diff
@@ -1121,7 +1058,7 @@ definition {
 				portalVersionOutput = "${portalVersionOutput}");
 		}
 
-		task ("View OAuth application scope") {
+		task ("View OAuth2 application scope") {
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase:1061]: I think we should be consistent: is OAuth 2 and application name or just type indicator, later you use the wording "Oauth 2" but here it's "OAuth2"

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]
Finished my review, see comments about the change request.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase (hunk 5 above)
Comment (weller0246) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase:1061]: You are right, I switched **OAuth2** to **OAuth 2** everywhere in the test case file.

**Comment by weller0246** [comment]
@gaborlovasdotcom  I resolved all your comments, please review again if you have the time

**Comment by gaborlovasdotcom** [comment]
> @gaborlovasdotcom I resolved all your comments, please review again if you have the time

Hi @weller0246 , I checked and there are still 6 occurrences for "OAuth2 ".

**Comment by weller0246** [comment]
Resolved and pushed!

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:oauth2:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by weller0246** [comment]
Yep, I rebase my branch and re run the test batch

**Comment by weller0246** [comment]
ci:test:security:nocompile

**Comment by weller0246** [comment]
ci:test:security:nocompile

**Comment by weller0246** [comment]
Fixed conflict

**Comment by gaborlovasdotcom** [comment]
ci:test:oauth2:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Review by gaborlovasdotcom — APPROVED** [review]
LGTM

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**